examples/dogpile_caching/__init__.py

```diff
@@ -58,3 +58,65 @@
           local_session_caching.py
 
 """
+
+from .._utils import DirectoryExamples
+
+
+REQUIREMENTS: DirectoryExamples = {
```

Comment:
I'd rather just add to the README simple steps for how to install dependencies and run the examples, the automation steps here make the examples look less accessible .

the examples here are mainly intended to be read and understood; running them should be a secondary concern and someone who is running them should want to read documentation how to do so, since that's why these are examples.

Reply:
Do you think this could be disclaimed with something like the following on `__Init__` files and dropping the payload to the bottom of the file ?

    # PLEASE IGNORE THE FOLLOWING LINES
    # THEY ARE FOR SQLALCHEMY'S CI SYSTEMS ONLY
    REQUIREMENTS: DirectoryExamples = {
    
This payload exists to test/prep the Examples for CI testing.  Because the examples are all non-standard in design/function/requirements, there needs to be a machine readable payload *somewhere* that can be easily mapped to each test.  They could even be in a `_sqla_maintainers.py` file with a big "IGNORE ME!" message in it.

Reply:
if it's for testing then it should be in the test/ folder, we have an examples suite that can be expanded